server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -55,12 +56,16 @@
 import org.apache.zookeeper.KeeperException;
 
 import com.google.common.net.HostAndPort;
+import java.util.concurrent.TimeUnit;
+import org.apache.accumulo.core.conf.Property;
 
 public class GarbageCollectWriteAheadLogs {
   private static final Logger log = Logger.getLogger(GarbageCollectWriteAheadLogs.class);
 
   private final Instance instance;
   private final VolumeManager fs;
+  private final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
```

Comment:
Are you suggesting I use a ConcurrentHashMap?  I thought SimpleGarbageCollector was designed to be one thread out of the [run](https://github.com/apache/accumulo/blob/1.6/server/gc/src/main/java/org/apache/accumulo/gc/SimpleGarbageCollector.java#L513) method.